webcompat/templates/privacy.html

```diff
@@ -5,25 +5,73 @@
+        </p>
+        </div>
+    </article>
+</section>
+<section class="grid">
+    <article class="grid-row">
+      <div class="grid-cell x3">
+        <h2 class="headline-1 highlight">Reports</h2>
+        <h3 class="headline-2 ">Public Reports</h3>
+        <p>
+            All reports are publicly accessible. Please do not include any confidential or personal information in the content of your report.
+        </p>
+        <p> 
+            If you choose to upload a screenshot or other image, it will be publicly accessible. Please do not include any confidential or personal information in the screenshot.
+        </p>
+        <p>
+            The User Agent HTTP header that your browser sends will be recorded in the bug report, if you choose to report a bug, but is hidden by default.
+        </p>
+        
+        <h3 class="headline-2">Reporting Anonymously</h3>
+        <p>
+            If you choose the anonymous option, the only information that we’ll receive is the content of your report.
+        </p>
+        
+        <h3 class="headline-2">Reporting via GitHub</h3>
+        <p>
+            If you choose the GitHub option, webcompat.com will receive your GitHub username and avatar.  You can revoke our access to this information at any time from the <a href="https://github.com/settings/applications" title="link to Githubs application settings">"Applications" section of your profile page</a>.
+        </p>
+        </div>
+    </article>
+</section>
+<section class="grid">
+    <article class="grid-row">
+      <div class="grid-cell x3">
+        <h2 class="headline-1 highlight">Logs &amp; Cookies</h2>
+        <h3 class="headline-2">Website Logs</h3>
+        <p>
+            When a bug report is filed on webcompat.com we log your IP address, the time at which the bug was filed, and the URI of the bug report. This data may be used to investigate potential security issues and spam efforts. These logs are only kept for 14 days, after which they are deleted automatically.
+        </p>
+        
+        <h3 class="headline-2">Cookies &amp; Analytics</h3>
+        <p>
+            If you log into our site using your GitHub user account, webcompat.com will use cookies to record your session information.  The cookie will expire when the session ends, i.e., when you click "logout".
+        </p>
+        <p>
+            Webcompat.com also uses Google Analytics to track usage statistics.
+        </p>
+          </div>
+    </article>
+</section>
+<section class="grid">
+    <article class="grid-row">
+      <div class="grid-cell x3">
+        <h2 class="headline-1 highlight">Contact</h2> 
+        <p>
+                You can find all contact information on <a href="./contact" title="our contact page">our contact page</a>.
```

Comment:
Small little nit: Double indentation here.

Reply:
We'll gonna run a linter through all of that on the rebase branch later on.